config/ponyfm.php

```diff
@@ -54,4 +54,15 @@
 
     'use_powered_by_footer' => env('USE_POWERED_BY_FOOTER', true),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Cache Duration
+    |--------------------------------------------------------------------------
+    |
+    | Duration in minutes for tracks to be stored in cache.
+    |
+    */
+
+    'cache_duration' => 1440,
```

Comment:
`cache_duration` is a very generic name but this is a value for a very specific cache. `track_file_cache_duration` or something would be a more accurate name for it.